spec/features/personas/sign_in_as_persona_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 feature "Sign In as Persona" do
   around do |example|
-    Capybara.app_host = "https://#{ENV["PLACEMENTS_HOST"]}"
+    Capybara.app_host = "https://#{ENV["CLAIMS_HOST"]}"
```

Comment:
this is a quick fix for the persona test -- I'm actually splitting it into two different tests (one for claims, one for placements in the next PR, but I just want to get the current_user stuff in so we can all use it for now.